src/app/shared/utils/consts.ts

```diff
@@ -188,6 +190,19 @@ export const Consts = {
                         name: 'unity',
                         rest: true,
                         ssh: false,
+                        cli: false,
+                        smis: false,
+                        extra: false
+                    }
+                },
+                {
+                    label: "VNX",
```

Comment:
could you please hold the VNX dictionary entry till we merge the vnx driver in delfin

Reply:
Done. removed the entry for now